config/routes.rb

```diff
@@ -1,3 +1,5 @@
 Rails.application.routes.draw do
+  resources :tutorials, only: :index
```

Comment:
Will create route helpers for the `tutorials_controller`'s `index` action: `tutorials_path` and `tutorials_url`.

Useful tip: run `bin/rails routes` to see a list of every known route (and their helper prefixes) in the application.